Balances plastanium window integrity

TheBonded opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Changes plastanium windows integrity to 1200 instead of 200, which leaves it with 200 more integrity than reinforced plasma windows. Also increases damage_deflection to 21 to match reinforced plasma windows
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
Reinforced plasma windows should not be stronger than plastanium glass windows, as titanium is stronger than iron, and that is the only difference between the two alloys. This pr changes that, making plastanium glass windows stronger than reinforced plasma windows.
<!-- Normal plasma windows should not be stronger than plasma+titanium glass windows. This changes that, while still keeping them at reasonable values. -->

## Changelog
:cl:
balance: Rebalanced Plastanium windows integrity and damage_deflection
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

TheBonded committed:
Change some window integrity values

Changes plastanium windows integrity to 350 instead of 200, and normal plasma windows to 250 instead of 300
 
Why this is good for the game:
Normal plasma windows should not be stronger than plasma+titanium glass windows. This changes that, while still keeping them at reasonable values

TheBonded committed:
Changes plastanium window integrity

Changes the integrity to 1200 and undoes nerf to plasma window

msgerbs commented:
Does it make sense to buff the damage_deflection of it as well/instead of an integrity buff? I think it might. At a certain point, integrity doesn't really matter but I think making it actually not take damage at all has a bigger impact on the impression of it being "tough". See: The change that was made to reinforced glass so it couldn't just be smashed by any random item.

TheBonded commented:
> Does it make sense to buff the damage_deflection of it as well/instead of an integrity buff? I think it might. At a certain point, integrity doesn't really matter but I think making it actually not take damage at all has a bigger impact on the impression of it being "tough". See: The change that was made to reinforced glass so it couldn't just be smashed by any random item.

I would definitely do this as well as my current changes however I have no idea how to do so.

msgerbs commented:
Change `damage_deflection = 11` to whatever value feels balanced (the next major damage threshold is 15 damage, from welders, surgical saws, etc.)

TheBonded committed:
Changes plastanium windows

Increases damage_deflection to match reinforced plasma windows at 21, and increased integrity from 200 to 1200 (200 higher than reinforced plasma windows)

TheBonded commented:
> Change `damage_deflection = 11` to whatever value feels balanced (the next major damage threshold is 15 damage, from welders, surgical saws, etc.)

Ah, thanks! I misread the plastanium as plasma and assumed plastanium had no damage_deflection stat. I changed it to match reinforced plasma windows, as this is supposed to act as a direct upgrade to that.

Barhandar commented:
Plasteel is stronger than plastitanium (titanium and plastitanium being reskins of regular walls with more laser reflection, not r-walls) last time I checked. Also, this is a buff to syndie stuff.

msgerbs commented:
Ok, why does plastitanium exist if it's just all around shittier than r-walls while needing mostly the same material?

Paxilmaniac commented:
Because it's glass

Fikou commented:
plastitanium walls are only deflecting stuff because of wall code being bad, nukie shuttle has good ones

msgerbs commented:
So what I'm getting out of this is: Unfuck plastitanium walls at the same time, and buffing nukie shuttle windows is not a bad thing because them having weak-ass windows is probably an accident.

CRITAWAKETS commented:
yeah i think the military-grade material should be stronger than some random industrial plating

i think plastitanium walls are as easy to decon as regular walls because nobody ever worked on deconstruction sprites.

Barhandar commented:
>Ok, why does plastitanium exist if it's just all around shittier than r-walls while needing mostly the same material?

To have fancy syndie black/red decorations.

TheBonded commented:
~test~ -test-
